### ISSUE TITLE
docs: User Management needs a summary of priviliges for roles

### ISSUE THREAD

**Issue by anand-chandrashekar** [scylladb/scylladb-cloud-doc-issues#18]
I would like to report an issue on page https://cloud.docs.scylladb.com/master/access-management/user-management

### Problem
Could we please provide details about what options are available for each role?

###  Suggest a fix
A table with available roles (Admin, Maintainer, Viewer) as columns and priviliges as rows (Create, delete).

**Comment by tzach** [comment]
@mixellent please advise

**Comment by mixellent** [comment]
@anand-chandrashekar did this request come from a specific customer?
Also, is there a specific privilege for a certain action that you are missing? Or your request is to have a more clear representation per role of which actions are allowed?

**Comment by anand-chandrashekar** [comment]
My request is that we summarise in an html table for consistency and readability.

**Comment by mixellent** [comment]
@annastuchlik Any thoughts on this?

**Comment by tzach** [comment]
I think a table is a good way to represent premissions
For example (draft, data might be wrong)
<google-sheets-html-origin><style type="text/css"></style>
  | View Cluster | Create CLuster | Update Cluster |   | Billing | User Mgmt | Secuirty Features | SSO
-- | -- | -- | -- | -- | -- | -- | -- | --
Admin | Yes | Yes | Yes |   | Yes | Yes | Yes | Yes
Maintainer | Yes | No | Yes |   | No | No | No | No
Viewer | Yes | No | No |   | No | No | No | No
SSO |   |   |   |   | ? | ? | ? | Yes

**Comment by annastuchlik** [comment]
@mixellent I agree that a table/diagram/other would be helpful. But I don't have full insight into permissions, so the page will have to be maintained by the dev team (once they add an action, they document who can perform that action).
